src/main/java/seedu/ddd/logic/commands/ListCommand.java

```diff
@@ -1,6 +1,7 @@
 package seedu.ddd.logic.commands;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.ddd.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
```

Comment:
Could you rename `MESSAGE_PERSONS_LISTED_OVERVIEW` to `MESSAGE_CONTACTS_LISTED_OVERVIEW` (or whatever makes sense)?